projects/packages/search/src/inline-search/class-inline-search.php

```diff
@@ -7,6 +7,8 @@
 
 namespace Automattic\Jetpack\Search;
 
+require_once __DIR__ . '/class-search-highlighter.php';
```

Comment:
I think there is autoloading. Discussed a bit here: https://github.com/Automattic/jetpack/tree/e6e7ae1698f111ab42ce6b44520871eff04a50ba/projects/packages#package-autoloading